samples/apps/research-assistant-react/README.md

```diff
@@ -0,0 +1,88 @@
+# ARA - AutoGen Research Assistant
+
+ARA is an Autogen-powered AI research assistant that can converse with you to help you conduct research, write and execute code, run saved skills, learn new skills by demonstration, and adapt in response to your interactions.
+
+Project Structure:
+
+- _backend/_ code for the web api, served by FastAPI.
+- _frontend/_ code for the webui, built with Gatsby and Tailwind
+
+## Getting Started
+
+Install requirements:
+
+```bash
+cd backend
+pip install -r requirements.txt
+```
+
+AutoGen requires access to an LLM. Please see the [AutoGen docs](https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints) on how to configure access to your LLM. We recommend setting the `OAI_CONFIG_LIST` environment variable to point to your LLM config file.
+
+Also, if you plan to use bing search, set the `BING_API_KEY`
+
+```bash
+export OAI_CONFIG_LIST=/path/to/OAI_CONFIG_LIST
+export BING_API_KEY=<your bing api key>
+```
+
+### Build and Run Frontend UI
+
+```bash
+cd frontend
+npm install
+npm run build
+```
+
+The command above will build the frontend ui and copy the build artifacts to the backend folder. Note that you may have to run `npm install --force` to force resolve some peer dependencies.
+
+Run the web ui:
+
+```bash
+cd backend
+uvicorn main:app --reload --port 8000
+```
+
+Navigate to <http://localhost:8000/> to view the web ui.
+
+To update the web ui, navigate to the frontend directory, make changes and rebuild the ui.
+
+## Capabilities
```

Comment:
Highlight how to switch agent workflow. Add example of how to add a new workflow.

Reply:
We have not tested the code paths when we switch the workflow. So we probably don’t want to highlight that?

Reply:
I think we should be? @samershi what do you think?

Reply:
What other agent workflows do we have built-in at the moment?

Reply:
We do not have any others. The planner agent is commented out, and very untested. The Group Chat agent is in another branch, and... needs work.

Reply:
@samershi, IMO that workflow testing will block the PR. We can add it as an issue for the future iterations of the RA?

Reply:
I'd just remove all mention of the Planner workflow... comments and otherwise. It's not tested. We abandoned it quickly back in September. Having it here adds work on testing something we already opted not to use. We can add in the ability to switch workflows in a later PR, when we add GroupChat.